test/coffeescript.coffee

```diff
@@ -30,7 +30,7 @@ describe 'coffeescript', ->
 				throw new MyError()
 			catch e
 
-			expect(e).to.have.deep.property('constructor.name', 'MyError')
+			expect(e.constructor.name).to.be.equal('MyError')
```

Comment:
Also I wonder how this deep prop test used to work, I think the deep prop is not about the dotted prop names, it's about "having prop x on any level of depth":

* https://www.webpackbin.com/bins/-KpVhsIGTJtdnE5GoBAR
* http://chaijs.com/api/bdd/#method_deep

What we want instead is `nested` http://chaijs.com/api/bdd/#method_nested